app/MMB/src/scenes/timeline/event/DescriptionComment.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Translation } from '@kiwicom/mobile-localization';
+import { Text, StyleSheet, Color } from '@kiwicom/mobile-shared';
+
+export type DescriptionCommentType = {|
+  description: string,
+  comment: string,
```

Comment:
Please always accept translation types instead of strings. Only this way we can be sure everything is translated properly.